glibc.yaml

```diff
@@ -34,7 +34,7 @@ environment:
       - bison
       - flex
       - grep
-      - python3
+      - python-3.11 # undo this
```

Comment:
This is just to trick `wolfictl text` into ignoring a cycle it thinks it has. We're not bumping the epoch, so the package won't get rebuilt with this dep, and we can undo this in a followup PR (also not bumping epoch) to avoid any confusion.